src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -4,693 +4,689 @@
-          long-summary: This range must not overlap with any Subnet IP ranges. For example, 10.0.0.0/16.
-        - name: --vnet-subnet-id
-          type: string
-          short-summary: The ID of a subnet in an existing VNet into which to deploy the cluster.
-        - name: --workspace-resource-id
-          type: string
-          short-summary: The resource ID of an existing Log Analytics Workspace to use for storing monitoring data. If not specified, uses the default Log Analytics Workspace if it exists, otherwise creates one.
-
-    examples:
-        - name: Create a Kubernetes cluster with an existing SSH public key.
-          text: az aks create -g MyResourceGroup -n MyManagedCluster --ssh-key-value /path/to/publickey
-        - name: Create a Kubernetes cluster with a specific version.
-          text: az aks create -g MyResourceGroup -n MyManagedCluster --kubernetes-version 1.12.6
-        - name: Create a Kubernetes cluster with a larger node pool.
-          text: az aks create -g MyResourceGroup -n MyManagedCluster --node-count 7
-""".format(sp_cache=AKS_SERVICE_PRINCIPAL_CACHE)
-
-helps['aks delete'] = """
-    type: command
-    short-summary: Delete a managed Kubernetes cluster.
+type: command
+short-summary: Create a new managed Kubernetes cluster.
+parameters:
+  - name: --generate-ssh-keys
+    type: string
+    short-summary: Generate SSH public and private key files if missing.
+  - name: --service-principal
+    type: string
+    short-summary: Service principal used for authentication to Azure APIs.
+    long-summary: If not specified, a new service principal is created and cached at $HOME/.azure/aksServicePrincipal.json to be used by subsequent `az aks` commands.
+  - name: --skip-subnet-role-assignment
+    type: bool
+    short-summary: Skip role assignment for subnet (advanced networking).
+    long-summary: If specified, please make sure your service principal has the access to your subnet.
+  - name: --client-secret
+    type: string
+    short-summary: Secret associated with the service principal. This argument is required if `--service-principal` is specified.
+  - name: --node-vm-size -s
+    type: string
+    short-summary: Size of Virtual Machines to create as Kubernetes nodes.
+  - name: --dns-name-prefix -p
+    type: string
+    short-summary: Prefix for hostnames that are created. If not specified, generate a hostname using the managed cluster and resource group names.
+  - name: --node-count -c
+    type: int
+    short-summary: Number of nodes in the Kubernetes node pool. After creating a cluster, you can change the size of its node pool with `az aks scale`.
+  - name: --node-osdisk-size
+    type: int
+    short-summary: Size in GB of the OS disk for each node in the node pool. Minimum 30 GB.
+  - name: --kubernetes-version -k
+    type: string
+    short-summary: Version of Kubernetes to use for creating the cluster, such as "1.7.12" or "1.8.7".
```

Comment:
These Kubernetes version strings were just updated in #8690. Could you update them in this PR as well?